extensions/extensions-helm/README.md

```diff
@@ -10,50 +13,66 @@ Each aiSSEMBLE Helm chart should contain the following
 **Note**: The unit tests are built with https://github.com/helm-unittest/helm-unittest#install and it is required to 
 install both Helm and the plugin locally to run this module.
 
+
 ## Leveraging Extensions Helm
+
 After an aiSSEMBLE module's Helm migration has been completed, the following steps can be taken in a downstream project 
 to leverage the new Helm pattern for the aiSSEMBLE module at hand:
 * Add the aiSSEMBLE helm repo to your local repos: `helm repo add ghcr.io https://UPDATE_WITH_HELM_REPO`
 
+
 ## For legacy aiSSEMBLE Projects
-* Delete the directory holding the module's existing Helm charts if they have already been generated:
-  * delete `<project-name>-deploy/src/main/resources/apps/<module-name>`
-  * Note: if any custom overrides/changes were made to any existing Helm artifacts, it is crucial to preserve these 
-    changes, prior to deleting this directory, so that they may be applied in the new Helm pattern
-* Update the deploy profile in the project's deploy pom file:
-  * in `<project-name>-deploy/pom.xml` replace `<profile><module-name>-deploy</profile>` with 
-    `<profile><module-name>-deploy-v2</profile>`, assuming the profile name corresponding to the new Helm pattern is 
-    noted with a "-v2" suffix
-* Rebuild the project
-* In `<project-name>-deploy/src/main/resources/apps/<module-name>/Chart.yaml`, update the `version` field to the 
-  corresponding aiSSEMBLE version
-* In the project's Tiltfile, make the following modifications:
-  * add the following lines above the module's `docker_build()` call:
-    * `local('helm dep update <project-name>-deploy/src/main/resources')`
-    * `local('helm dependency build <project-name>-deploy/src/main/resources/apps/<module-name>') `
-  * Below the module's corresponding `k8s_resource()` call, add the following line to manually build the docker image 
-    for the migrated module:
-    * `local('docker build -f <project-name>-docker/<project-name>-<module-name>-docker/target/Dockerfile -t <project-name>-<module-name>:latest <project-name>-docker/<project-name>-<module-name>-docker/ --build-arg VERSION_AISSEMBLE=<current aiSSEMBLE version>') `
-      * Note: this step is not always necessary, and only needed when the chart is dependent on an image being built in 
-        a downstream project
-
-After making these changes, the migrated module in the aiSSEMBLE project can properly leverage the new Helm pattern, 
-defined by the Extensions Helm module.
+
+Follow the instructions in the [_Kubernetes Artifacts Upgrade_](https://boozallen.github.io/aissemble/aissemble/current/containers.html#_kubernetes_artifacts_upgrade)
+section of the _Path to Production > Container Support_ documentation page to update older projects to the new Extensions Helm baseline approach.
 
 ## Developing with Extension Helm
```

Comment:
I: The ticket comments have a lot more detail about how we arrived at this approach with breadcrumbs to different pieces of the puzzle.